src/components/Resources/ResourcePage.js

```diff
@@ -38,7 +48,30 @@ function ResourcePage({ matchProps }) {
             </Link>
             <Typography color="textPrimary">{resource.title}</Typography>
           </Breadcrumbs>
-          <h2>{resource.title}</h2>
+          <Typography variant="h2" gutterBottom>
+            {resource.title}
```

Comment:
I think we should deconstruct these attributes from `resource` for clarity. i.e.:

```javascript
const { title, subtitle, url } = resource;

...

<Typography variant="h2">
  { title }
</Typography>
```

Reply:
+++

Reply:
Thank again for the tips! This is now destructured: ff568ee